lib/src/rendering/overflow_view.dart

```diff
@@ -165,23 +164,23 @@ class RenderOverflowView extends RenderBox
     }
 
     while (child != lastChild) {
-      child = childParentData.nextSibling;
+      child = childParentData.nextSibling!;
       childParentData = child.parentData as OverflowViewParentData;
       childParentData.offstage = true;
     }
 
-    if (unrenderedChildCount > 0) {
+    if (unRenderedChildCount > 0) {
       // We have to layout the overflow indicator.
-      final RenderBox overflowIndicator = lastChild;
+      final RenderBox? overflowIndicator = lastChild;
```

Comment:
If we have at least one unRenderedChildCount, then lastChild cannot be null. We could then do 

``` dart
final RenderBox overflowIndicator = lastChild!;
```

instead